app/components/sorting-key.js

```diff
@@ -0,0 +1,31 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  tagName: 'th',
+  classNameBindings: ['isSorted:active', 'isAsc:asc', 'isDesc:desc'],
+  isSorted: (function() {
```

Comment:
https://github.com/artisan-tattoo/assistant-frontend/pull/54/files#diff-d8fbe1b980499a71bd6949c7876eb1d5R16 this style is considered better